dateparser/conf.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+
+class Settings(object):
+    PREFER_DATES_FROM_FUTURE = None
```

Comment:
Ok. But let's make this "current_period", "past" and "future" strings (not objects) to make it easier to read from the file later.